actioncable/lib/action_cable/channel/base.rb

```diff
@@ -160,7 +161,10 @@ def perform_action(data)
         action = extract_action(data)
 
         if processable_action?(action)
-          dispatch_action(action, data)
+          payload = {channel_class: self.class.name, action: action, data: data}
```

Comment:
Can you add spaces and after the brackets?

`{ channel_class: self.class.name, action: action, data: data }`